assets/i18n/strings_fr.i18n.json

```
@@ -0,0 +1,604 @@
+    "loginSuccessful": "Connexion réussie",
+    "authSocketException": "Il y a un problème avec le réseau. Veuillez vérifier la connexion.",
+    "snsLogin": "Connexion SNS"
+  },
+  "stored": {
+    "saved": "Sauvegardé",
+    "savedPosts": "Publications sauvegardées",
+    "postSaved": "Publication sauvegardée",
+    "postSavedMessage": "Vous pouvez voir les publications sauvegardées dans Ma page"
+  },
+  "notification": {
+    "likeNotificationsTitle": "Mentions J’aime",
```

Biome (2.3.13)

[error] The key notification was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)